Sluggo/Models/Codables/WritePinnedTicketRecord.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  WritePinnedTicketRecord.swift
+//  Sluggo
+//
+//  Created by Isaac Trimble-Pederson on 5/5/21.
+//
+
+import Foundation
+
+struct WritePinnedTicketRecord: Codable {
+    let ticketID: Int
+}
```

Comment:
We have some codables whose write records live within the same file. Can you move those out to different files?

On a related note, would you be willing to unify all filenames as [codable type]Record?

Reply:
( this is lower priority and I'm fine with merging this in if need be )